node-notes-cli/app.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const notebook = require('./data.json');
+
+const command = process.argv[2];
+const ID = notebook.nextId;
+const notes = notebook.notes;
+
+function updateNotebook() {
+  fs.writeFile('data.json', JSON.stringify(notebook, null, 2), err => {
+    if (err) throw err;
+  });
+}
+
+if (command === 'read') {
+  for (const property in notes) {
+    console.log(`${property}: ${notes[property]}`);
+  }
+} else if (command === 'create') {
+  notes[ID] = process.argv[3];
+  notebook.nextId++;
+
+  updateNotebook();
```

Comment:
When you call `updateNotebook` you should pass in the `noteBook` object to be written to the file.